projects/vdk-core/src/vdk/api/job_input.py

```diff
@@ -361,3 +361,22 @@ def get_execution_properties(self) -> dict:
                 SELECT id, {pa__job_start_ts_expr} FROM mytable
         """
         pass
+
+    @abstractmethod
+    def skip_remaining_steps(self) -> None:
+        """
+        :return:
+
+        Usage:
+            job_input.skip_remaining_steps()
+
+            Raises an internal exception that signalises to the VDK runtime that all remaining steps should be skipped
```

Comment:
nit: "Raises an internal exception" seems like epxosing an implementation detail. I think say "Singalizes to the VDK runtime that all remaining steps should be skipped.
```suggestion
            Signalizes to the VDK runtime that all remaining steps should be skipped
```